lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -596,6 +608,32 @@ LLDBMemoryReader::getFileAddressAndModuleForTaggedAddress(
   return {{file_address, module}};
 }
 
+bool LLDBMemoryReader::readRemoteAddressImpl(
+    swift::remote::RemoteAddress address, swift::remote::RemoteAddress &out,
+    std::size_t size) {
+  assert((size == 4 || size == 8) &&
+         "Only 32 or 64 bit architectures are supported!");
```

Comment:
Suggestion: early return `false` here if this condition doesn't hold. Then you don't have to do the `else return false; return true;` at the end.